src/main/java/fi/aalto/cs/apluscourses/utils/FileDateFormatter.java

```diff
@@ -4,80 +4,24 @@
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.attribute.FileTime;
-import java.time.Clock;
 import java.time.ZoneOffset;
 import java.time.ZonedDateTime;
-import java.time.temporal.ChronoUnit;
-import java.util.Arrays;
-import java.util.List;
-import java.util.Locale;
 
 public class FileDateFormatter {
```

Comment:
Nit: this class could be renamed to `FileUtils` or something to allow other utility functions to be added here in the future. As such as it is currently, the purpose of this class seems a bit questionable to me.